app/sass/settings/_liberator.scss

```diff
@@ -18,3 +19,6 @@ $inuit-widths-columns: (
 
 $base-font-stack: Georgia, Cambria, "Times New Roman", Times, serif;
 
+$inuit-enable-margins--large: true;
+$inuit-enable-responsive-margins: true;
+$inuit-generate-spacing-alias: s;
```

Comment:
If this line is removed, it will produce error in console saying "unbound variable $inuit-generate-spacing-alias on line 81 in project/root/directory/node_modules/inuit-responsive-spacing/_trumps.spacing-responsive.scss" and Gulp sass task won't work. The letter "s" assigned to this variable is just randomly chosen.